### PR TITLE
Use ClassName for adapter tests yaml filters

### PR DESCRIPTION
**Pull request by EricDahlvang**
Fixes https://github.com/microsoft/botbuilder-dotnet/issues/4448

### CODE REVIEW

File: FunctionalTests/Microsoft.Bot.Builder.FunctionalTests/FacebookChatTests.cs [hunk 1]
```diff
@@ -10,6 +10,8 @@
 
 namespace Microsoft.Bot.Builder.FunctionalTests
 {
+    // NOTE: this class name is referenced in the Functiona Tests yaml file.
```
Comment (BruceHaley) [inline FunctionalTests/Microsoft.Bot.Builder.FunctionalTests/FacebookChatTests.cs:13]: Typo: "Functional"

File: FunctionalTests/Microsoft.Bot.Builder.FunctionalTests/SlackClientTests.cs [hunk 2]
```diff
@@ -12,10 +12,12 @@
 
 namespace Microsoft.Bot.Builder.FunctionalTests
 {
+    // NOTE: this class name is referenced in the Functiona Tests yaml file.
```
Comment (BruceHaley) [inline FunctionalTests/Microsoft.Bot.Builder.FunctionalTests/SlackClientTests.cs:15]: Typo: "Functional"

File: FunctionalTests/Microsoft.Bot.Builder.FunctionalTests/TwilioNumberTests.cs [hunk 3]
```diff
@@ -14,6 +14,8 @@
 
 namespace Microsoft.Bot.Builder.FunctionalTests
 {
+    // NOTE: this class name is referenced in the Functiona Tests yaml file.
```
Comment (BruceHaley) [inline FunctionalTests/Microsoft.Bot.Builder.FunctionalTests/TwilioNumberTests.cs:17]: Same typo

File: FunctionalTests/Microsoft.Bot.Builder.FunctionalTests/WebexClientTests.cs [hunk 4]
```diff
@@ -14,10 +14,12 @@
 
 namespace Microsoft.Bot.Builder.FunctionalTests
 {
+    // NOTE: this class name is referenced in the Functiona Tests yaml file.
```
Comment (BruceHaley) [inline FunctionalTests/Microsoft.Bot.Builder.FunctionalTests/WebexClientTests.cs:17]: Same typo

Comment (EricDahlvang) [reply FunctionalTests/Microsoft.Bot.Builder.FunctionalTests/WebexClientTests.cs]: oops, thanks @BruceHaley

**Pull request closed without merging**